### PR TITLE
fix: use includes over contains

### PR DESCRIPTION
**Pull request by daibhin**
## Changes

Use includes to check substring. `contains` is throwing a weird error as reported in https://github.com/PostHog/posthog-js/pull/1272#issuecomment-2191304203

|Before|After|
|----|----|
| Screenshot 2024-06-26 at 11 20 36 | Screenshot 2024-06-26 at 11 20 42 |

|Before|After|
|----|----|
| Screenshot 2024-06-26 at 11 20 16 | Screenshot 2024-06-26 at 11 21 09 |

### CODE REVIEW

File: patches/rrweb@2.0.0-alpha.13.patch [hunk 1]
```diff
@@ -224,20 +224,22 @@ index da2c103fe6b1408a5996f0eb3bf047571e434cc2..f5b060c7e0728a3e2c6cf62b01d97282
...
      if (tagName === 'link' && inlineStylesheet) {
 -        const stylesheet = Array.from(doc.styleSheets).find((s) => {
+-            return s.href === n.href;
++        const href = n.href
 +        let stylesheet = Array.from(doc.styleSheets).find((s) => {
-             return s.href === n.href;
++            return s.href === href;
          });
-+        if (!stylesheet && n.href.contains('.css')) {
++        if (!stylesheet && href.includes('.css')) {
```
Comment (marandaneto) [inline patches/rrweb@2.0.0-alpha.13.patch:240]: probably not an issue but includes is case sensitive
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

**Review by marandaneto — APPROVED** [review]

File: patches/rrweb@2.0.0-alpha.13.patch (hunk 1 above)
Comment (daibhin) [reply patches/rrweb@2.0.0-alpha.13.patch:240]: Should be ok for our use case. I'm not sure `.CSS` is a valid file suffix

File: patches/rrweb@2.0.0-alpha.13.patch [hunk 2]
```diff
@@ -224,20 +224,22 @@ index da2c103fe6b1408a5996f0eb3bf047571e434cc2..f5b060c7e0728a3e2c6cf62b01d97282
...
      }
      if (tagName === 'link' && inlineStylesheet) {
 -        const stylesheet = Array.from(doc.styleSheets).find((s) => {
+-            return s.href === n.href;
++        const href = n.href
 +        let stylesheet = Array.from(doc.styleSheets).find((s) => {
-             return s.href === n.href;
++            return s.href === href;
          });
-+        if (!stylesheet && n.href.contains('.css')) {
```
Comment (marandaneto) [inline patches/rrweb@2.0.0-alpha.13.patch:238]: not part of the mozilla spec at least https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String so probably depends on browser yeah

**Comment by pauldambra** [comment]
Lgtm

**Pull request merged**